### PR TITLE
feat(attraqt): Added Attraqt documentation

### PR DESCRIPTION
**Pull request by Kagaminara**
Added documention on how to enable Attraqt datasource for Magento2.

- [Configuration](https://deploy-preview-491--heuristic-almeida-1a1f35.netlify.app/docs/magento2/search-engine#attraqt)
- [Environment variable](https://deploy-preview-491--heuristic-almeida-1a1f35.netlify.app/docs/reference/environment-variables#attraqt)

### CODE REVIEW

File: docs/magento2/search-engine.mdx
Comment (abailleulfc) [inline docs/magento2/search-engine.mdx:234]: Why not a simple message like in other pages ?  (e.g. maintenance mode, password fields, etc)

```suggestion
> _Since version 2.19_
```

File: docs/magento2/search-engine.mdx
Comment (abailleulfc) [inline docs/magento2/search-engine.mdx:252]: https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to
```suggestion
You can find your search API key by log in to your XO console, and navigating
```

A link to the XO console's targeted page may be good too (is it a generic URL ?)

File: docs/magento2/search-engine.mdx
Comment (abailleulfc) [inline docs/magento2/search-engine.mdx:257]:
```suggestion
The script should be very quick to execute, and will let you now what needs your
```

File: docs/magento2/search-engine.mdx
Comment (abailleulfc) [inline docs/magento2/search-engine.mdx:325]:
```suggestion
If you are using the default theme or the Chocolatine theme, the search bar
```

File: docs/reference/environment-variables.mdx
Comment (abailleulfc) [inline docs/reference/environment-variables.mdx:233]: what about the directly link here too ?

File: docs/magento2/search-engine.mdx
Comment (dpobel) [inline docs/magento2/search-engine.mdx:247]: same remarks as in https://gitlab.com/front-commerce/front-commerce/-/merge_requests/1502/diffs#note_1119089954

File: docs/magento2/search-engine.mdx
Comment (dpobel) [inline docs/magento2/search-engine.mdx:243]:
```suggestion
Using your Attraqt's search API key, you can execute this command from your
```

File: docs/magento2/search-engine.mdx
Comment (dpobel) [inline docs/magento2/search-engine.mdx:253]:
```suggestion
You can find your search API key by login in into the XO console, and navigating
to: `Search > API Keys`
```

and it would be nice if possible to put a link

File: docs/magento2/search-engine.mdx
Comment (dpobel) [inline docs/magento2/search-engine.mdx:299]:
```suggestion
Then, in your `.env` file, you need to define the `FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY` environment variable:

```bash title=.env
FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY=yourapikeyvalue
```
```

File: docs/reference/environment-variables.mdx
Comment (dpobel) [inline docs/reference/environment-variables.mdx:233]:
```suggestion
`FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY`: your Attraqt search API key, found in
your XO console under `Search > API Keys`
```

and same remark if a link can be added

**Review by dpobel — COMMENTED** [review]
and the MR should be against `next` not `dev`

File: docs/magento2/search-engine.mdx
Comment (Kagaminara) [reply docs/magento2/search-engine.mdx:252]: Typo noted !

> A link to the XO console's targeted page may be good too (is it a generic URL ?)

Unfortunately it isn't.

File: docs/magento2/search-engine.mdx
Comment (Kagaminara) [reply docs/magento2/search-engine.mdx:325]: Also fixed the line I copy-pasted it from 👍

File: docs/reference/environment-variables.mdx
Comment (Kagaminara) [reply docs/reference/environment-variables.mdx:233]: Same as above, those are not generic URLs

File: docs/magento2/search-engine.mdx
Comment (Kagaminara) [reply docs/magento2/search-engine.mdx:247]: Yes, this will definitely need to be updated once we have a script MR is merged

File: docs/reference/environment-variables.mdx
Comment (Kagaminara) [reply docs/reference/environment-variables.mdx:233]: I can add a link to the console, but unfortunately can't do anything more than that.

I'll add this one at  least though.

Comment (Kagaminara) [reply docs/reference/environment-variables.mdx]: I added a link to the console though as Damien suggested below.

**Review by abailleulfc — APPROVED** [review]

**Comment by Kagaminara** [comment]
I'll wait for the script MR to be merged before merging this, in case I have changes to make.

**Pull request merged**
